### PR TITLE
feat(kit): `InputDate` add min/max support for native input date

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?

part of #1358

### CODE REVIEW

File: projects/kit/components/input-date/native-date/native-date.component.ts [hunk 1]
```diff
@@ -26,6 +30,14 @@ export class TuiNativeDateDirective {
             : '';
     }
 
+    get max(): string {
+        return this.host.max.toString('YMD', '-');
```
Comment (waterplea) [inline projects/kit/components/input-date/native-date/native-date.component.ts:34]: I think that's just `toJSON()`.

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/components/input-date/examples/6/index.html [hunk 2]
```diff
@@ -3,5 +3,18 @@
...
 >
-    <tui-input-date formControlName="testValue">Choose a date</tui-input-date>
+    <tui-input-date
+        formControlName="testValue"
+        class="tui-space_bottom-4"
+    >
+        Choose a date
+    </tui-input-date>
+
+    <tui-input-date
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/input-date/examples/6/index.html:13]: Cypress test fails.
See https://github.com/Tinkoff/taiga-ui/actions/runs/5059051359/jobs/9080189963?pr=4519

```
    (Attempt 1 of 2) native data picker
    1) native data picker
  1 passing (9s)
  1 failing
  1) InputDate
       native data picker:
CypressError: `cy.type()` can only be called on a single element.
Your subject contained 2 elements.
```

**Review by nsbarsukov — APPROVED** [review]

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

File: projects/demo/src/modules/components/input-date/examples/6/index.html [hunk 3]
```diff
@@ -3,5 +3,18 @@
...
+        formControlName="testValue2"
+        [max]="max"
+        [min]="min"
```
[the remark this answers is not in the document]
Comment (nsbarsukov) [reply projects/demo/src/modules/components/input-date/examples/6/index.html:16]: https://github.com/Tinkoff/taiga-ui/assets/35179038/840e1eca-006e-4bf6-98e3-f4e291f24ff7

[the remark this answers is not in the document]
Comment (vladimirpotekhin) [reply projects/demo/src/modules/components/input-date/examples/6/index.html]: Safari doesn't show min/max in native picker ( we decided to remove the example from the demo, but keep the functionality for the future

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**